research/src/pages/components/slider.research.parts.mdx

```diff
@@ -5,6 +5,8 @@ showInMenu: false
 layout: ../../layouts/ComponentLayout.astro
 ---
 
+import '../../styles/research.css'
```

Comment:
I'm not sure where to put this import, because it _could_ live here. But the styles are written quite generically so that it could be used elsewhere too.

Maybe in the future we could move the HTML into its own component. Non-blocking, but want to field for ideas :)

Reply:
Yeah. I just want to touch as little as possible, since that's best way to try water.

Reply:
Also regarding generic styles. `.true` and `.false` was super annoying to check where it was used. I have to look for `"true `, `true"` and ` true ` to validate that I did not miss some hidden corner. Can we rename this to `value-true`/`value-false` eventually?

Reply:
Great idea! And you're right, we don't need to worry about that right now since it's such a hidden away page.